src/neuroconv/tools/nwb_helpers/_dataset_configuration.py

```diff
@@ -104,10 +103,11 @@ def get_default_dataset_io_configurations(
     known_dataset_fields = ("data", "timestamps")
     for neurodata_object in nwbfile.objects.values():
         if isinstance(neurodata_object, DynamicTable):
-            dynamic_table = neurodata_object  # For readability
 
-            for column in dynamic_table.columns:
-                candidate_dataset = column.data  # VectorData object
+            dynamic_table = neurodata_object  # For readability
+            for column in (*dynamic_table.columns, dynamic_table.id):
```

Comment:
Here, we add the ids of the tables to the set that we catch for setting the DataIO.